browser/index.js

```diff
@@ -29,7 +27,7 @@ document.addEventListener("DOMContentLoaded", async () => {
   }
 
   const dht = new KadDHT({
-    protocolPrefix: "/archaeologist-service",
+    protocolPrefix: import.meta.env.VITE_DHT_PROTOCOL_PREFIX || "/archaeologist-service",
```

Comment:
the purpose of the custom protocol prefix is to try to limit the nodes found to just your local nodes (through the signaling server)-- doesn't fully work but helps (seems like some other nodes still sneak through